src/index.html

```diff
@@ -25,17 +25,20 @@
         <div class="container">
             <h1 class="pt-4 text-center"><i class="fa-solid fa-medal text-primary"></i>LeaderBoard</h1>
         </div>
+        <div id="alert" class="alert alert-success" role="alert"></div>
         <div class="container d-flex flex-row justify-content-between mt-4 p-2">
-            <div class="row col-lg-6 mt-4">
-                <div class="row d-flex flex-row justify-content-between">
-                    <div class="col">
+            <div class="row col-lg-6 mt-4 mb-2">
+                <div class="d-flex flex-row justify-content-between">
+                    <h2>Recent Score</h2>
+                    <button type="button" id="getter" class="btn btn-primary">Refresh</button>
+                    <!-- <div class="col">
                         <h2>Recent Score</h2>
                     </div>
                     <div class="col">
                         <button type="button" id="getter" class="btn btn-primary">Refresh</button>
-                    </div>
+                    </div> -->
```

Comment:
- [ ] I would recommend removing the commented code to make  the codebase cleaner.